src/illustrations/Chest.jsx

```diff
@@ -105,8 +124,15 @@ const Chest = (props) => {
                 l ` + (251.33336 + widthPlus) + `, 43.66669
                 l -1.66667, 33
                 l -232.00002, -10.33331
-                l 0, 181.23331z`}/> 
+                l 0, 181.23331z`}/>
```

Comment:
Hmm, why do you use the template literals without utilizing the interpolation? (I guess for multiline strings?) You should write these like this: 
`` `l ${251.33336 + widthPlus}, 43.66669` ``, there's no need (or point) to concatenate strings at all. Read more here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals